tests/regression_tests/filter_translations/test.py

```diff
@@ -0,0 +1,94 @@
+from itertools import permutations
+from openmc import mesh
+import numpy as np
+
+import openmc
+import pytest
+
+from tests.testing_harness import PyAPITestHarness
+
+
+@pytest.fixture
+def model(request):
+
+    model = openmc.model.Model()
+
+    fuel = openmc.Material()
+    fuel.set_density('g/cm3', 10.0)
+    fuel.add_nuclide('U235', 1.0)
+    zr = openmc.Material()
+    zr.set_density('g/cm3', 1.0)
+    zr.add_nuclide('Zr90', 1.0)
+    model.materials.extend([fuel, zr])
+
+    box1 = openmc.model.rectangular_prism(10.0, 10.0)
+    box2 = openmc.model.rectangular_prism(20.0, 20.0, boundary_type='reflective')
+    top = openmc.ZPlane(z0=10.0, boundary_type='vacuum')
+    bottom = openmc.ZPlane(z0=-10.0, boundary_type='vacuum')
+    cell1 = openmc.Cell(fill=fuel, region=box1 & +bottom & -top)
+    cell2 = openmc.Cell(fill=zr, region=~box1 & box2 & +bottom & -top)
+    model.geometry = openmc.Geometry([cell1, cell2])
+
+    model.settings.batches = 5
+    model.settings.inactive = 0
+    model.settings.particles = 1000
+
+    translation = np.array((10, -5, 0))
+
+    llc = np.array([-9, -9, -9])
+    urc = np.array([9, 9, 9])
+
+    mesh_dims = (3, 4, 5)
+
+    filters = []
+
+    # un-translated meshes
+    reg_mesh = openmc.RegularMesh()
+    reg_mesh.dimension = mesh_dims
+    reg_mesh.lower_left = llc
+    reg_mesh.upper_right = urc
+
+    filters.append(openmc.MeshFilter(reg_mesh))
+
+    recti_mesh = openmc.RectilinearMesh()
+    recti_mesh.x_grid = np.linspace(llc[0], urc[0], mesh_dims[0])
+    recti_mesh.y_grid = np.linspace(llc[1], urc[1], mesh_dims[1])
+    recti_mesh.z_grid = np.linspace(llc[2], urc[2], mesh_dims[2])
+
+    filters.append(openmc.MeshFilter(recti_mesh))
+
+    llc -= translation
+    urc -= translation
```

Comment:
This actually ends up changing `llc` and `urc` for the _previously_ defined `reg_mesh` filter, so it has the same lower left and upper right as the translated one. I noticed this because the results_true file indicates that tallies 1 and tallies 3 do not have the same results.

Reply:
Ah, very true! Thanks for catching that.